src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,19 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    @Override
+    public boolean test(Candidate candidate) {
+        return candidate.getAge() >= 35
```

Comment:
You should use constants for your checks

```suggestion
public class CandidateValidator implements Predicate<Candidate> {
    private static final int  PRESIDENT_MIN_AG = 35;
    
    @Override
    public boolean test(Candidate candidate) {
        return candidate.getAge() >= PRESIDENT_MIN_AGE;
```